### DISCUSSION TITLE
Meeting 3 Agenda

### DISCUSSION THREAD

**Discussion by volcaniqueo** [bounswe/bounswe2023group1#33] [Meeting Agendas]
Let's add the entries for possible discussion topics for our upcoming meeting at 15 March.

**Comment by kubraaksux** [comment]
Our communication plan seems so necessary and urgent

**Reply by kubraaksux** [reply]
Also maybe this should be done before 15th

**Comment by volcaniqueo** [comment]
Labels & Usage Principles

**Comment by kubraaksux** [comment]
meeting with our contact and maybe a new meeting with suzan hoca will be necessary after this and our weekly

**Comment by volcaniqueo** [comment]
Finalizing details about Revisioner role & discuss about its efficiency

**Comment by kubraaksux** [comment]
using slack and notion

**Comment by kubraaksux** [comment]
Base features and requirements page details
